nvcf/nvcf_test.ipynb

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Can we add a curl test to the README along with this python-based notebook validation?